packages/block-library/src/query-title/index.php

```diff
@@ -22,7 +45,14 @@ function render_block_core_query_title( $attributes ) {
 	}
 	$title = '';
 	if ( $is_archive ) {
-		$title = get_the_archive_title();
+		$show_prefix = isset( $attributes['showPrefix'] ) ? $attributes['showPrefix'] : true;
+		if ( ! $show_prefix ) {
+			add_filter( 'get_the_archive_title', 'block_query_title_filter_archive_title' );
+			$title = get_the_archive_title();
+			add_filter( 'get_the_archive_title', 'block_query_title_filter_archive_title' );
```

Comment:
I assume this one was supposed to be `remove_filter`?

Reply:
Yes!